tester/main.cpp

```diff
@@ -35,61 +38,58 @@ typedef struct cat_t {
 
 int main(int argc, char const *argv[]) {
 	// common options
-	vc::Option includeCustomUtils("custom_include", "-include", "utils.hpp");
-	vc::Option includeCustomUtilsDir("custom_includeDir", "-I", "../include");
-	vc::Option includePolybenchUtils("polybench_include", "-include", "polybench.c");
-	vc::Option includePolybenchUtilsDir("pb_Dir", "-I", polybench_source + "/utilities");
-	vc::Option optimizationLevel("O","-O","3");
-	vc::Option standardCxx("std","-std=","c++11");
-	std::list<vc::Option> default_options = {
-		standardCxx,
-		includePolybenchUtils,
-		includeCustomUtils,
-		optimizationLevel,
-		includePolybenchUtilsDir,
-		includeCustomUtilsDir,
+	const vc::opt_list_t default_options {
+                vc::make_option("-include utils.hpp"),
```

Comment:
not sure if leaving a space between `-include` and `utils.hpp` is going to make it work. I remember having troubles in certain configurations. I'm going to double check later.